### PR TITLE
Add Hotjar support

### PR DESCRIPTION
**Pull request by mateimicu**
Added support [HotJar](https://www.hotjar.com/) support

### CODE REVIEW

File: exampleSite/config.toml [hunk 1]
```diff
@@ -57,8 +57,11 @@ rootURL = "//matomo.example.com/"
 [params.fathomAnalytics]
 rootURL = "//fathom.example.com/"
 
+[params.hotjarAnalytics]
+hjid = 1
```
Comment (DanielRuf) [inline exampleSite/config.toml:61]: I think this should be disabled by default.

File: layouts/partials/analytics/hotjar.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<script>
+    (function(h,o,t,j,a,r){
+        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+        h._hjSettings={hjid:'{{ .Site.Params.matomoAnalytics.hjid }}',hjsv:6};
```
Comment (DanielRuf) [inline layouts/partials/analytics/hotjar.html:4]: Did you forget to add the new param or is this the wrong config key?

https://github.com/mateimicu/kiss/blob/add_hotjar_support/exampleSite/config.toml#L53

**Comment by mavogel** [comment]
@DanielRuf can we merge this? :)

**Comment by mateimicu** [comment]
I deployed this branch to my personal website and it works

**Review by DanielRuf — APPROVED** [review]

**Pull request merged**